website/content/api-docs/secret/databases/oracle.mdx

```diff
@@ -130,3 +130,10 @@ list the plugin does not support that statement type.
   base64-encoded semicolon-separated string, a serialized JSON string array, or
   a base64-encoded serialized JSON string array. The `{{name}}` value will be
   substituted. If not provided defaults to a generic drop user statement.
+
+- `rotation_statements` `(list: [])` – Specifies the database statements to be
+  executed to rotate the password for a given username. Must be a
```

Comment:
The part after `Must be a ...` is confusing to me. It reads like it must be all of one class or all of another.

Reply:
I am not sure I understand the confusion. This is just copied from the other dbs docs. Do you have a suggestion for improving the wording?

Reply:
I think it is because it sounds like it must be all of the first 3 types or the last.  Can it be any combination and any number of them?

Reply:
Maybe I am assuming too much, but it would be really strange to me for the `rotation_statements` field to require a mix of formats.

Reply:
Fair enough. Should pull this bit out to a partial so that it is not duplicated?

E.g: https://github.com/hashicorp/vault/tree/9ffea8aee557da186bfa142af6fb217c2b967699/website/content/partials